lib/Clownface.js

```diff
@@ -126,18 +197,39 @@ class Clownface {
     return Clownface.fromContext(context)
   }
 
+  /**
+   * Creates graph pointer to one or more blank node(s)
+   * @param {null|string|string[]} [values] blank node identifiers (uses generated if undefined of null)
```

Comment:
Ah, I realized that all the chain methods also accept Clownface itself. 
Should I change like this where applicable? @bergos 

```suggestion
   * @param {null|string|string[]} [values] blank node identifiers (uses generated if undefined of null)
   * @param {null|string|string[]|Clownface|Clownface[]} [values] blank node identifiers (uses generated if undefined of null)
```

Reply:
Jup, should be also included